### ISSUE TITLE
PDF Text gets blurry after a while

### ISSUE THREAD

**Issue by ibehnam** [ahrm/sioyek#491]
<img width="1262" alt="image" src="https://user-images.githubusercontent.com/58621210/202277702-edd847bd-ef02-4119-b088-511b53d89e36.png">

It affects individual files. The file in the other tab (same window) looks fine.

**Comment by ibehnam** [comment]
I've set `linear_filter` to be 1.

**Comment by ahrm** [comment]
Does it still happen with experimental builds here: https://github.com/hexomancer/sioyek/releases/tag/v0.31.409? (also maybe disable `linear_filter` when using these new builds, it should not be necessary anymore ).

**Comment by ibehnam** [comment]
Yes it also happens with experimental builds. Disabling `linear_filter` helps.

**Comment by ahrm** [comment]
Do you have any idea on what causes this? (for example does it happen after clicking on a link, resizing the screen, etc.)

**Comment by ibehnam** [comment]
Not really.  It mostly happens when sioyek has been in the background for too long.

**Comment by ahrm** [comment]
Does this still happen in the latest versions of sioyek?

**Comment by ibehnam** [comment]
I haven't tried v2.0 yet so maybe others can comment on that.
